Graph/Axis.php

```diff
@@ -62,106 +42,67 @@ public function __construct()
         $this->labels = new Labels();
     }
 
-    /**
-     * @return Title
-     */
-    public function getTitle()
+    public function getTitle(): Title
     {
         return $this->title;
     }
 
-    /**
-     * @param Title $title
-     */
-    public function setTitle($title)
+    public function setTitle(Title $title): void
     {
         $this->title = $title;
     }
 
-    /**
-     * @return bool
-     */
     public function getOpposite(): bool
     {
         return $this->opposite;
     }
 
-    /**
-     * @param bool $opposite
-     */
-    public function setOpposite(bool $opposite)
+    public function setOpposite(bool $opposite): void
     {
         $this->opposite = $opposite;
     }
 
-    /**
-     * @return bool
-     */
     public function isCrosshair(): bool
```

Comment:
I doubt that this works, as the 'crosshair' (and pretty much the majority of properties) are nullable. The PHPDoc type-hint is wrong. You probably need to run rector again to correct the return types.

Did you run the unit tests to verify all was working?

Reply:
If so... we may have false positives...